SignalShareExtension/ShareViewController.swift

```diff
@@ -41,57 +42,65 @@ class ShareViewController: UINavigationController, SAELoadViewDelegate {
 
         setupEnvironment()
 
+        // TODO:
 //        [UIUtil applySignalAppearence];
```

Comment:
FYI I'm moving UIUtil as part of my upcoming PR if you want to ignore this for now.

Reply:
Great.